### DISCUSSION TITLE
ImportError: cannot import name 'PatchDiscriminator' from 'monai.networks.nets'

### DISCUSSION THREAD

**Discussion by jaykshirsagar05** [Project-MONAI/MONAI#8033] [General]
I am trying this tutorial: [2dlm](https://github.com/Project-MONAI/tutorials/tree/main/generation/2d_ldm) but facing this error: `ImportError: cannot import name 'PatchDiscriminator' from 'monai.networks.nets'`

Using 1.3.2 Monai

**Comment by KumoLiu** [comment]
Hi @jaykshirsagar05, the new component will be include in the core in 1.4. You can try latest container or weekly version for now.
https://pypi.org/project/monai-weekly/
https://hub.docker.com/r/projectmonai/monai

Thanks.
